src/benchmark_api_internal.h

```diff
@@ -10,10 +12,29 @@
 
 #include "benchmark/benchmark.h"
 #include "commandlineflags.h"
+#include "string_util.h"
 
 namespace benchmark {
 namespace internal {
 
+struct Base10HumanReadableFormat {
```

Comment:
this doesn't belong in the public interface of the internal api.  there are existing string format files that it might belong to.